lib/eventboss/runner.rb

```diff
@@ -25,6 +25,7 @@ def launch
         Eventboss::DevelopmentMode.setup_infrastructure(queues) if config.development_mode?
 
         begin
+          validate_client!(client, config)
```

Comment:
Will we keep this forever or at somepoint (after finishing migration) we can drop this validation?